File: backend-go/internal/modules/system/entity/sys_job_run_log.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysJobRunLog struct {
	ID           common.Int64String `gorm:"column:id;primaryKey" json:"id"`
	JobID        common.Int64String `gorm:"column:job_id" json:"jobId"`
	JobName      string             `gorm:"column:job_name" json:"jobName"`
	Status       string             `gorm:"column:status" json:"status"`
	StartedAt    *time.Time         `gorm:"column:started_at" json:"startedAt"`
	FinishedAt   *time.Time         `gorm:"column:finished_at" json:"finishedAt"`
	CostMs       common.Int64String `gorm:"column:cost_ms" json:"costMs"`
	ErrorMessage *string            `gorm:"column:error_message" json:"errorMessage"`
}

func (SysJobRunLog) TableName() string { return "sys_job_run_log" }

File: backend-python/app/modules/system/entity/sys_job_run_log.py
from datetime import datetime

from sqlalchemy import BigInteger, DateTime, String
from sqlalchemy.orm import Mapped, mapped_column

from app.modules.system.entity.base import Base


class SysJobRunLog(Base):
    __tablename__ = "sys_job_run_log"

    id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
    job_id: Mapped[int] = mapped_column(BigInteger)
    job_name: Mapped[str] = mapped_column(String(64))
    status: Mapped[str] = mapped_column(String(16))
    started_at: Mapped[datetime] = mapped_column(DateTime)
    finished_at: Mapped[datetime | None] = mapped_column(DateTime, nullable=True)
    cost_ms: Mapped[int] = mapped_column(BigInteger)
    error_message: Mapped[str | None] = mapped_column(String(512), nullable=True)
